RSSReader/Presenter/FeedPresenter.m

```diff
@@ -0,0 +1,39 @@
+//
+//  FeedPresenter.m
+//  RSSReader
+//
+//  Created by Владислав Станкевич on 19.11.20.
+//
+
+#import "FeedPresenter.h"
+#import "NetworkManager.h"
+#import "RSSParser.h"
+
+@implementation FeedPresenter
+
+-(id)initWithArray:(NSMutableArray *)array {
+    if (self = [super init]) {
+        self.feedItemArray = array;
+    }
+    return self;
+}
+
+- (void)loadNewsWiithCompletion:(void (^)(BOOL))completion {
+    NetworkManager *networkManager = [NetworkManager new];
+    self.networkManager = networkManager;
```

Comment:
Single responsibility principle (SRP) violation. Your function is supposed to load smth. Please, move network manager creation and configuration out to the separate call. Lazy loading implementation would be preferable.